app/views/helpers/TimelineEvents.scala

```diff
@@ -18,22 +18,38 @@ package views.helpers
 
 import java.time.ZonedDateTime
 
+import config.featureFlags.SfusConfig
+import javax.inject.Inject
 import models.declaration.notifications.Notification
+import models.declaration.submissions.Submission
 import play.api.i18n.Messages
 import play.twirl.api.Html
+import views.html.components.upload_files_partial_for_timeline
 
-case class TimelineEvent(title: String, dateTime: ZonedDateTime, content: Option[Html]) extends Ordered[TimelineEvent] {
+case class TimelineEvent(title: String, dateTime: ZonedDateTime, content: Option[Html])
 
-  def compare(that: TimelineEvent): Int =
-    if (dateTime == that.dateTime) 0
-    else if (dateTime.isAfter(that.dateTime)) -1
-    else 1
-}
+class TimelineEvents @Inject()(sfusConfig: SfusConfig, uploadFilesPartialForTimeline: upload_files_partial_for_timeline) {
+  def apply(submission: Submission, notifications: Seq[Notification])(implicit messages: Messages): Seq[TimelineEvent] = {
+    val sortedNotifications =
+      notifications
+        .map(notification => notification.copy(dateTimeIssued = notification.dateTimeIssuedInUK))
```

Comment:
Not sure what the purpose of this reassignment is, the ordering would not be affected in any way :thinking:

Reply:
From what I read the normalisation we are doing by using `ZonedDateTime.withZoneSameInstant` could potentially affect the order if the source ZonedDateTime instances in `Notification` have different time zones.

But tbh, I wasn't sure, so I went for this solution just to be safe.